dev/NavigationView/NavigationView_InteractionTests/NavigationViewTests.cs

```diff
@@ -558,6 +558,23 @@ public void AlwaysShowHeaderTest()
                     headerVisibilityCheckbox.Check();
                     Wait.ForIdle();
                     VerifyElement.Found("Home as header", FindBy.Name);
+
+                    // PaneDisplayMode and Top option were added on RS5, so just run the next tests if we are not using RS4 Style
+                    if (!testScenario.IsUsingRS4Style)
```

Comment:
Not sure if this if the best place to add this test. Please let me know if you have suggestions of better places.